superset-frontend/src/common/components/index.tsx

```diff
@@ -43,6 +45,11 @@ export const MenuItem = styled(AntdMenu.Item)`
   > a {
     text-decoration: none;
   }
+
+  &.ant-menu-item {
+    height: 30px;
+    line-height: 30px;
```

Comment:
I guess it was another PR with the same CSS where we were discussing using the Theme's typography size for this... same comment applies here :)

Reply:
Done